scipy/linalg/_solvers.py

```diff
@@ -150,7 +150,7 @@ def _solve_discrete_lyapunov_bilinear(a, q):
 
 def solve_discrete_lyapunov(a, q, method=None):
     """
-    Solves the discrete Lyapunov equation :math:`A'XA-X=-Q`.
+    Solves the discrete Lyapunov equation :math:`AXA^H - X = -Q`.
```

Comment:
True.  Now that @nmayorov mentioned the MATLAB doc, it does occur to me `Q` does not need to be necessarily Hermitian, though in most contexts, I have seen that to be the case.